.favorites.json

```diff
@@ -0,0 +1,26 @@
+[
```

Comment:
Don't think this file should be submitted

Reply:
will revert it .. in my next commit I had it on gitignore but on the other branch